packages/itwin/grouping-mapping-widget/src/widget/components/SortableHorizontalTile.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from "react";
```

Comment:
Missing copyright header.

Reply:
Huh, the git hook didn't run the script to insert it. Manually added it.